src/components/Landing/index.tsx

```diff
@@ -63,6 +80,32 @@ export default function Landing() {
     margin: '0 auto',
   }
 
+  const statsRowStyles: CSSProperties = {
+    borderTop: `1px solid var(--stroke-tertiary)`,
```

Comment:
**question**: can we use colors from `ThemeContext`? Might be worth grepping this PR for `var(` to find other areas we can use ThemeContext,too.